krun.py

```diff
@@ -413,32 +440,32 @@ def main():
     for vm_name, vm_info in config["VMS"].items():
         vm_info["vm_def"].set_platform(platform)
 
-    log_filename = attach_log_file(config_file)
+    # If the user has asked for resume-mode, the current platform must
+    # be an identical machine to the current one.
+    current = None
+    if args.resume:
+        if os.path.isfile(out_file):
+            current = util.read_results(out_file)
+            if not util.audits_same_platform(platform.audit, current["audit"]):
+                msg = """You have asked krun to resume an interrupted benchmark.
```

Comment:
You can use `+` operator so as to make the indentation look better. Or move it out into a global, where we are already at indent level 0. I prefer the former, as the latter ruins locality.